index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
- [ ] It Would be so much better if you explains `why` to use a more descriptive title. The explanation of the request you are asking for helps to understand why this change is needed and how to handle it properly. So please add a `Why`  explanation to your change required. 
- [ ] [OPTIONAL] Use a check box mark, this helps to organize the review and helps to check if the change requested was already addressed. 
- [ ] [OPTIONAL] Use emojis and/or gifs in the review. This makes the code review more human-friendly
😄 👍